### ISSUE TITLE
One container didn't got an update of the configuration

### ISSUE THREAD

**Issue by ghostbar** [deis/deis#3794]
So I had 3 web instances of an app. I updated the config with `deis config:set` and the instance `web.1` didn't got that update of the conf. I made it `null` and then again set it and `web.2` and `web.3` got the update but again not `web.1`.

I tried downscaling it to 0 and then back to 3, 2, even 1.

I scaled it up to 4 and that one did got the update in the configuration but again not 1.

There's a way to delete an specific container? I wanted `web.1` gone. Even after getting it to deploy another versions that one did not got the configuration updated.

**Comment by bacongobbler** [comment]
@ghostbar interesting. Can you tell us a bit more about this app? Have you previously deployed an application with the same name? I wonder if it's due to the fact that data plane images aren't cleaned up once an app is destroyed. What does `journalctl -u <appname>_v<num>.web.1` show on that host? You could try a `docker rmi -f \`docker images -aq\`` from that host and try `deis ps:restart web.1` to force the host to pull the image from the registry. The envvars are baked directly into the image so it must be that the host has something different than the image you'd expect.

**Comment by ghostbar** [comment]
So it was a randomly created ID for the app. It was in about v57. It was the password for a backing service and it got updated on all of the containers except that one.

I deleted the app, created another one and that one did got the updates even tho I scaled it up to 5 instances (we have a cluster with 5 machines) and they got the updates on the config.

I tried with `journalctl` but it seems since I deleted the app the logs got deleted as well :(
